pipeline.libsonnet

```diff
@@ -56,6 +56,7 @@
     gcsObject():: artifact('gcs/object', 'gcs'),
     githubFile():: artifact('github/file', 'github'),
     gitlabFile():: artifact('gitlab/file', 'gitlab'),
+    gitRepo():: artifact('git/repo', null),
```

Comment:
I'm new to this code, so apologies if this is obvious to other folks.  Does the null here deserve a comment since it appears to be the only one, or perhaps some additional logic in `artifact` above to check for null instead of passing a null kind?

Reply:
I'm pretty new as well, but it looks like kind is not part of the artifact format anymore: https://spinnaker.io/reference/artifacts-with-artifactsrewrite/#the-artifact-format

I've found https://github.com/spinnaker/spinnaker/issues/3740 which was fixed by https://github.com/spinnaker/sponnet/commit/a1f852a721cb63a4137bf26800433c68ef8e7c74. The discussion on the issue suggests that that attribute was required by deck a few years ago, then it was deprecated by https://github.com/spinnaker/spinnaker/issues/3820 (and probably removed at some point afterwards).

I can submit a PR to remove the kind attribute. from all artifact types.

Reply:
Yeah, I agree it makes sense to remove kind.  It would be great to see the removal in a separate PR and then this one can focus on git/repo and helm charts.  Thanks!

Reply:
@dbyron-sf I've submitted https://github.com/spinnaker/sponnet/pull/32, please take a look.